include/aspect/velocity_boundary_conditions/interface.h

```diff
@@ -59,6 +59,8 @@ namespace aspect
          * Initialization function. This function is called once at the
          * beginning of the program after parse_parameters is run and after
          * the SimulatorAccess (if applicable) is initialized.
+         *
+         * @deprecated Use initialize(types::boundary_id boundary_id) instead.
          */
         virtual
         void
```

Comment:
Is this a mistake? I don't see any other `initialize` function with this set of arguments.